docs/src/test-sharding-js.md

```diff
@@ -26,6 +26,25 @@ Now, if you run these shards in parallel on different jobs, your test suite comp
 
 Note that Playwright can only shard tests that can be run in parallel. By default, this means Playwright will shard test files. Learn about other options in the [parallelism guide](./test-parallel.md).
 
+
+## Balancing Shards
```

Comment:
Sounds like you need a rebaseline.

Reply:
Rebase applied, thanks for the  feedback. :)